scripts/launch-tests.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
This script is useless if we fix test/cli in parallel mode.
Much easier to learn:
```
mocha test/dir -p --watch
```
then
```
scripts/launch-tests.js dir --watch
```

Reply:
I disagree with you: I don't want to watch my test files all the time, and worse: not all the tests should be run in parallel.
The part where I agree is that the tests should be improved so that they can be run in parallel.

Reply:
I really don't we should create a wrapper for this.
This will break
```
npm test -- test/app*
```
And there is very little to no improvements to test script.

Reply:
This won't break it: it's still possible by doing `npm test app`.
I still agree: we should **definitely** improve the CLI tests. If we can't run them in parallel, then we won't and my solution stands.